### DISCUSSION TITLE
Update to 15.0.4 only can login local Network

### DISCUSSION THREAD

**Discussion by CommanderDer** [inventree/InvenTree#7634] [Q&A]
Hello
please excuse my bad english. I have the following problem. I have updated my inventory to version 15.0.4. Previously I could access the server on the local network as well as via cloudflare. Since the update it only works locally and the login page is also displayed externally but the login fails. I suspect it has something to do with the base_URL. However, since the update I can no longer set this via the web interface -> Settings. I also tried to set it via config.yaml but despite restarting, this URL is not accepted.  I noticed that in the config_template.yaml on Git
# site_url: 'http://localhost:8000' is there and in mine it is # base_url: 'http://localhost:8000' .
what is the difference between base_url and site_url?

**Comment by SchrodingersGat** [comment]
You will need to rename `base_url` to `site_url` in your config file. Then it should work OK

**Reply by CommanderDer** [reply]
So I have now changed base_url to site_url. Then I restarted but it still doesn't work. The entry will also not be accepted. On the website under Settings there is still the IP of my internet provider and not the URL of cloudflare. What I also read is that Environment Variables are still in config.yaml. But where exactly can I set INVENTREE_SITE_URL, for example?

**Comment by SchrodingersGat** [comment]
You will need to provide more details, such as how you have installed InvenTree, and how you are "serving" / proxying the setup.

Before that, have a close read of this section of the documentation: https://docs.inventree.org/en/stable/start/config/#server-access

**Reply by CommanderDer** [reply]
Thank you. I noticed again that changing the site_url in the config.yaml file doesn't do anything. By setting the URL using the command `inventree config:set INVENTREE_SITE_URL=https://my-url.com` I can now log in again.

**Reply by SchrodingersGat** [reply]
@CommanderDer yes the environment variables take priority over anything specificed in the configuration file
